File: 1468-check-if-n-and-its-double-exist/check-if-n-and-its-double-exist.py
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        lookup = Counter(arr)

        for num in arr:
            if (num != 0 and (num * 2) in lookup):
                return True
            if (num == 0 and lookup[num] > 1):
                return True
        return False

        

File: 1468-check-if-n-and-its-double-exist/check-if-n-and-its-double-exist.js
var checkIfExist = function(arr) {
    let lookup = {}

    for (let num of arr) {
        if (!(num in lookup)) lookup[num] = 0
        lookup[num] ++ 
    }

    for (let num of arr) {
        if (num === 0 && lookup[num] > 1 )  {
            return true
        } 
        if (num != 0 && (num * 2) in lookup) {
            return true
        } 
    }    
    return false
};